modules/git/command.go

```diff
@@ -103,18 +103,19 @@ func (c *Command) SetDescription(desc string) *Command {
 	return c
 }
 
-// AddArguments adds new git argument(s) to the command. Each argument must be safe to be trusted.
-// User-provided arguments should be passed to AddDynamicArguments instead.
-func (c *Command) AddArguments(args ...CmdArg) *Command {
+// AddTrustedArguments adds new git argument(s) to the command.
+// Each argument must be safe to be trusted, so only string literals should be passed.
+// User-provided arguments should be passed to AddUntrustedArguments instead.
+func (c *Command) AddTrustedArguments(args ...CmdArg) *Command {
```

Comment:
1. That's not true: `so only string literals should be passed.`
2. Why I didn't choose `AddTrustedArguments`: it could be misleading. For example, there are codes like:
    * `cmd.AddTrustedArguments("-c", CmdArg("user.name="+opts.Committer.Name)`: is the `CommiterName` trusted or not?
    * `cmd.AddTrustedArguments(CmdArg("--prefix=" + filepath.Base(strings.TrimSuffix(repo.Path, ".git")) + "/"))`: is the generated path trusted?

Reply:
2 is  just my worry about the readability. 

`Trusted` is a very strong word, it could lead to misunderstand more easliy.

And, using `AddTrustedArguments` could also have other problems (the comment below).